tables/atom.py

```diff
@@ -101,7 +101,7 @@ def _normalize_shape(shape) -> Shape:
     return tuple(SizeType(s) for s in shape)
 
 
-def _normalize_default(value, dtype):
+def _normalize_default(value: Any, dtype: np.dtype) -> np.ndarray:
```

Comment:
Here probably it is better to use DTypeLike.
In this case, by the way, we should bump the minimum numpy version to 1.20

Reply:
Good idea! 
I upgraded the minimum `numpy` version from `>=1.19` to `>=1.20`, and changed the annotation to `DTypeLike`.